lib/gettext-setup/version.rb

```diff
@@ -0,0 +1,3 @@
+module GettextSetup
+  VERSION = '0.24'
```

Comment:
I've been looking for a way to avoid maintenance commits to update the version prior to every release. I'm wondering if there's a way to have this constant be a little more dynamic in any way.

Backing up, what is the goal for adding this constant?